src/scripts.js

```diff
@@ -71,6 +73,19 @@ const bookARoom = (e) => {
     domUpdates.displayBookedMessage();
     // individualRoom.classList.add('white-shadow-transform');
 
+
+    // postBooking(roomToPost)
```

Comment:
Oops! Need to delete. I was trying to re-formatting but got caught in curly bracket hell.